app/src/main/java/org/mozilla/fenix/search/awesomebar/ShortcutsSuggestionProvider.kt

```diff
@@ -26,7 +29,12 @@ class ShortcutsSuggestionProvider(
     override val id: String = UUID.randomUUID().toString()
 
     private val settingsIcon by lazy {
-        AppCompatResources.getDrawable(context, R.drawable.ic_settings)?.toBitmap()
+        AppCompatResources.getDrawable(context, R.drawable.mozac_ic_settings)?.apply {
```

Comment:
how come this is being colored at runtime and not in the vector xml?

Reply:
The vector in AC has other colors (`mozac_ui_icons_fill`). Most of the components in AC are using also a tint besides the image resource, a tint that we pass from the client (see `BrowserMenuImageText` and other `BrowserMenuItem` menu items). In places where we use the resource directly, we have to tint them ourselves. (The overriding of `mozac_ui_icons_fill` proved to have other downsides due to having  the private theme.)